docs/self-managed/reference-architecture/manual/manual.md

```diff
@@ -0,0 +1,127 @@
+sidebar_label: Manual JAR
+description: "Camunda 8 Manual (Java) deployment Reference architecture home "
+---
+
+<!-- Moving target, may be renamed, etc. -->
+
+# Reference Architectures Overview: Manual JAR Deployment
+
+This section of the Camunda Deployment Reference Architectures provides guidance on deploying Camunda Platform as a standalone Java application. This deployment method is ideal for users who prefer manual deployment on bare metal servers or virtual machines (VMs), offering full control over the environment and configuration. It is particularly suited for scenarios with specific infrastructure requirements or highly customized setups.
+
+## Key Features
+
+- **Single application JAR**: Starting from Camunda 8.7, all core components (Zeebe, Tasklist, Operate, Optimize, and Identity) are bundled into a single JAR file. This simplifies deployment by reducing the number of artifacts to manage.
+- **Full Control**: Users are responsible for all aspects of deployment, including installation, configuration, scaling, and maintenance. This offers maximum flexibility for custom environments.
+
+Other deployment options, such as containerized deployments or managed services, might offer more convenience and automation. However, VM based deployment gives you the flexibility to tailor the deployment to your exact needs, which can be beneficial for regulated or highly customized environments.
+
+For documentation on the orchestration cluster, Web Modeler and Console separation, refer to [the documentation](/self-managed/reference-architecture/reference-architecture.md#orchestration-cluster-vs-web-modeler-and-console)
+
+## Reference implementations
+
+This section includes deployment reference architectures for manual setups:
+
+- [Aamazon EC2 deployment](/self-managed/setup/deploy/amazon/aws-ec2.md) - is a standard production setups with support for high availability.
+
+## Before You Start
+
+Before starting with the self-managed single JAR setup, consider evaluating your deployment platform options (Kubernetes / Docker) based on your familiarity with these platforms. Self-managing your deployment requires a solid understanding of infrastructure, networking, and application management. If you prefer a simpler and more managed solution, you may want to explore [our SaaS offerings](https://camunda.com/platform/). SaaS can significantly reduce maintenance efforts, allowing you to focus on your core business needs.
+
+## Limitations
+
+- The focus is on the orchestration cluster. This includes the single JAR compromised of Identity, Operate, Optimize, Tasklist, and Zeebe. AS well as the Connectors runtime.
+- General guidance and examples will focus on **unix** users but can be adapted by Windows users with the use of e.g. [WSL](https://learn.microsoft.com/en-us/windows/wsl/install) or included `batch` files.
+- The deployment of `Optimize` is still based on the old architecture and must run on a single node.
```

Comment:
```suggestion
- The deployment of Optimize is based on the old architecture and must run on a single node.
```

A link to the old architecture(?) could help here; this seems unclear to a new user.

Reply:
I'm not sure we have anything in the documentation that talks about the overall architecture at all so far ...

Reply:
I changed it to align with the aws-ec2 consideration mentioning

> The Optimize importer is not highly available and must only run once within the whole setup.

Reply:
as a result we could probably remove this from the `aws-ec2` page and just link the considerations on the `manual` page?